table/src/main/java/com/bazaarvoice/emodb/table/db/astyanax/AstyanaxTableDAO.java

```diff
@@ -1541,4 +1551,50 @@ static boolean isTableBlacklisted(Table table, Condition blackListTableCondition
         Map<String, Object> tableAttributes = table.getAttributes();
         return ConditionEvaluator.eval(blackListTableCondition, tableAttributes, new TableFilterIntrinsics(table));
     }
+
+    /*
+       The output of toString From ZonedDateTime will be one of the following ISO-8601 formats:
+       uuuu-MM-dd'T'HH:mm
+       uuuu-MM-dd'T'HH:mm:ss
+       uuuu-MM-dd'T'HH:mm:ss.SSS
+       uuuu-MM-dd'T'HH:mm:ss.SSSSSS
+       uuuu-MM-dd'T'HH:mm:ss.SSSSSSSSS
+       Note that the format used will be the shortest that outputs the full value of the time where the omitted parts are implied to be zero.
+
+       This helper method will force the Seconds and Milliseconds precision when serializing ZonedDateTime.
+       for example:
+       2017-01-01T07:01Z will always be serialized as 2017-01-01T07:01:00.000Z
+       2017-01-01T07:01:01Z will always be serialized as 2017-01-01T07:01:01.000Z
+     */
+    @VisibleForTesting
+    protected static String getMillisecondPrecisionZonedDateTime(Object dateTime) {
```

Comment:
seems like we accept parameters only `ZonedDateTime` type, why not change `Object` to it?